main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -55,6 +55,33 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     );
   };
 
+  var doAddColumnWithUrlOfMatchedEntities = function AddColumnWithUrlOfMatchedEntities (columnName,dismissDialog) {
+    promptForColumn(AddColumnWithUrlOfMatchedEntities);
```

Comment:
This will not work, because the callback you are supplying to the `promptForColumn` function cannot be the same as the outer function that calls `promptForColumn`. If you do that, you will run into the following problem (assuming that `promptForColumn` works properly):
* when the user first clicks on the menu entry to run the operation, `AddColumnWithUrlOfMatchedEntities` is called
* inside the body of `AddColumnWithUrlOfMatchedEntities`, `promptForColumn` is called
* once the user has supplied a column name, the success callback supplied to `promptForColumn` is called, which is `AddColumnWithUrlOfMatchedEntities` again
* inside the body of `AddColumnWithUrlOfMatchedEntities`, `promptForColumn` is called
* once the user has supplied a column name, the success callback supplied to `promptForColumn` is called, which is `AddColumnWithUrlOfMatchedEntities` again
* inside the body of `AddColumnWithUrlOfMatchedEntities`, `promptForColumn` is called
* once the user has supplied a column name, the success callback supplied to `promptForColumn` is called, which is `AddColumnWithUrlOfMatchedEntities` again
* inside the body of `AddColumnWithUrlOfMatchedEntities`, `promptForColumn` is called
* once the user has supplied a column name, the success callback supplied to `promptForColumn` is called, which is `AddColumnWithUrlOfMatchedEntities` again
* ...

so this is an infinite loop!